tree/dataframe/src/RLoopManager.cxx

```diff
@@ -116,6 +116,11 @@ void RLoopManager::RunTreeProcessorMT()
 /// Run event loop over one or multiple ROOT files, in sequence.
 void RLoopManager::RunTreeReader()
 {
+   if (fTree->IsA() == TClass::GetClass("TChain")) {
```

Comment:
Note that LoadTree is legal on a TTree and you may actually want to call it there too in case some of the friend trees are TChains.